platform/linux-generic/arch/arm/odp_atomic.h

```diff
@@ -28,4 +28,82 @@ do {							\
 
 #endif  /* CONFIG_DMBSTR */
 
+#include <limits.h>
```

Comment:
To be more consistent with project coding style I would move this include above `#ifdef CONFIG_DMBSTR` line.

Also, copyright message in this file can be updated.
```
/* Copyright (c) 2017-2021, ARM Limited
 * Copyright (c) 2017-2018, Linaro Limited
 * All rights reserved.
```

Reply:
Thanks Matias! I will make these changes